scripts/statistics.js

```diff
@@ -1,17 +1,39 @@
 'use strict';
 const bcd = require('..');
+const { RANGE_PREFIX_LTE: RANGE_PREFIX_LT } = require('../utils.js');
```

Comment:
I don't think we need to a export a single character to a utils file that pollutes our root directory even more. It is very vaguely named file and I don't think it is very useful to anyone. Please keep this inline and don't create this export.

Reply:
I’d like to fix this, but I don’t currently have a working PC I could commit this on.

Reply:
Thanks for the heads up! If you think you will get to in a few days, I can wait for that. 
If not, I can update this next week or so and then get it merged.

Reply:
This will now continue in https://github.com/mdn/browser-compat-data/pull/4583